model/Software/Classes/PackageGitoid.md

```diff
@@ -0,0 +1,23 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# PackageGitoid
+
+## Summary
+
+Preferred verification method to be used for a Package which represents an artifact that logically binds a number of single files together.
+
+## Description
+
+https://www.iana.org/assignments/uri-schemes/prov/gitoid Gitoid stands for [Git Object ID](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects) and a gitoid of type blob is a unique hash of a binary artifact. 
+
+Note that in addition to being used as a verification method, gitoid is also a reccomended ExternalIdentifier to be used for the package with the ExternalIdentifierType gitoid.
+
+## Metadata
+
+- name: PackageGitoid
+- SubclassOf: /Software/PackageVerification
+
+## Properties
+
+- gitoid
```

Comment:
So, the idea is that we have a verification class which is just a hash. Given that `gitoid` are equivalent with `swhid`, shall these be abstracted or shall there be another class with a single property?

Reply:
I agree `swhid` could also be used for package verification.  A consistent approach would be to add another class with a single property.